tests/test_basic_functionality.py

```diff
@@ -43,6 +43,14 @@ def empty_tree(layers: list[str]) -> LayeredConfigTree:
     return LayeredConfigTree(layers=layers)
 
 
+@pytest.fixture
+def getter_dict() -> NestedDict:
```

Comment:
Why is this fixture a dict rather than just being a LayeredConfigTree?

Reply:
We need access to the dict in the tests to test for equality because LayeredConfigTree doesn't have an equality operator.